test/JDBC/expected/BABEL-5186-vu-verify.out

```diff
@@ -0,0 +1,2460 @@
+BEGIN TRAN;
+SET XACT_ABORT OFF;
+INSERT INTO babel_5186_1_table_errTable VALUES (211);
+INSERT INTO babel_5186_1_table_errTable VALUES (212);
+SELECT * FROM non_existent_table;
+INSERT INTO babel_5186_1_table_errTable VALUES (213);
+COMMIT TRAN;
+GO
+~~ROW COUNT: 1~~
+
+~~ROW COUNT: 1~~
+
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: relation "non_existent_table" does not exist)~~
+
+
+SELECT * FROM babel_5186_1_table_errTable
+GO
+~~START~~
+int
+111
+112
+121
+122
+131
+132
+141
+142
+~~END~~
+
+
+BEGIN TRAN;
+SET XACT_ABORT OFF;
+INSERT INTO babel_5186_1_table_errTable VALUES (221);
+INSERT INTO babel_5186_1_table_errTable VALUES (222);
+EXEC('SELECT * FROM non_existent_table;');
+INSERT INTO babel_5186_1_table_errTable VALUES (223);
+COMMIT TRAN;
+GO
+~~ROW COUNT: 1~~
+
+~~ROW COUNT: 1~~
+
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: relation "non_existent_table" does not exist)~~
+
+
+SELECT * FROM babel_5186_1_table_errTable
+GO
+~~START~~
+int
+111
+112
+121
+122
+131
+132
+141
+142
+~~END~~
+
+
+BEGIN TRAN;
+SET XACT_ABORT OFF;
+INSERT INTO babel_5186_1_table_errTable VALUES (231);
+INSERT INTO babel_5186_1_table_errTable VALUES (232);
+SELECT non_existent_column FROM babel_5186_1_try_catch_table;
+INSERT INTO babel_5186_1_table_errTable VALUES (233);
+COMMIT TRAN;
+GO
+~~ROW COUNT: 1~~
+
+~~ROW COUNT: 1~~
+
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: column "non_existent_column" does not exist)~~
+
+
+SELECT * FROM babel_5186_1_table_errTable
+GO
+~~START~~
+int
+111
+112
+121
+122
+131
+132
+141
+142
+~~END~~
+
+
+BEGIN TRAN;
+SET XACT_ABORT OFF;
+INSERT INTO babel_5186_1_table_errTable VALUES (241);
+INSERT INTO babel_5186_1_table_errTable VALUES (242);
+EXEC('SELECT non_existent_column FROM babel_5186_1_try_catch_table;');
+INSERT INTO babel_5186_1_table_errTable VALUES (243);
+COMMIT TRAN;
+GO
+~~ROW COUNT: 1~~
+
+~~ROW COUNT: 1~~
+
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: column "non_existent_column" does not exist)~~
+
+
+SELECT * FROM babel_5186_1_table_errTable
+GO
+~~START~~
+int
+111
+112
+121
+122
+131
+132
+141
+142
+~~END~~
+
+
+-- Simple batch with nested transaction
+BEGIN TRAN;
+SET XACT_ABORT OFF;
+INSERT INTO babel_5186_1_table_errTable VALUES (311);
+BEGIN TRAN;
+INSERT INTO babel_5186_1_table_errTable VALUES (312);
+SELECT * FROM non_existent_table;
+COMMIT TRAN;
+INSERT INTO babel_5186_1_table_errTable VALUES (313);
+COMMIT TRAN;
+GO
+~~ROW COUNT: 1~~
+
+~~ROW COUNT: 1~~
+
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: relation "non_existent_table" does not exist)~~
+
+
+SELECT * FROM babel_5186_1_table_errTable
+GO
+~~START~~
+int
+111
+112
+121
+122
+131
+132
+141
+142
+~~END~~
+
+
+BEGIN TRAN;
+SET XACT_ABORT OFF;
+INSERT INTO babel_5186_1_table_errTable VALUES (321);
+BEGIN TRAN;
+INSERT INTO babel_5186_1_table_errTable VALUES (322);
+EXEC('SELECT * FROM non_existent_table;');
+COMMIT TRAN;
+INSERT INTO babel_5186_1_table_errTable VALUES (323);
+COMMIT TRAN;
+GO
```

Comment:
I would suggest adding `SELECT @@trancount` after every explicit transaction block test, just to confirm the transaction block is gone.

Reply:
Added